build/release-version.js

```diff
@@ -10,35 +10,41 @@ async function main() {
     return console.log('please specify a branch name');
   }
 
+  await configureGit();
   const exists = await checkIfBranchExists(branchName);
-  exists ? updateBranch() : createBranch();
+  exists ? await updateBranch(branchName) : await createBranch(branchName);
 
-  await push();
+  await push(branchName);
+}
+
+async function configureGit() {
+  await exec(`git config --global user.email "sandbox_JSUI@coveo.com"`);
+  await exec(`git config --global user.name "Github Action Bot"`);
 }
 
 async function checkIfBranchExists(branchName) {
-  try {
-    await exec(`git show-ref --verify --quiet refs/heads/${branchName}`);
-    return true;
-  } catch (e) {
-    console.log(`did not find a branch called: ${branchName}`);
-    return false;
-  }
+  const { stdout } = await exec(`git ls-remote --heads origin ${branchName}`);
+  return stdout !== '';
 }
 
 async function updateBranch(branchName) {
   console.log(`updating branch: ${branchName}`);
 
   await exec(`git checkout ${branchName}`);
-  await exec(`npm version patch`);
+  await bumpVersion('patch');
 }
 
 async function createBranch(branchName) {
   console.log(`creating branch: ${branchName}`);
 
   await exec(`git checkout -b ${branchName}`);
-  const newVersion = await getNewReleaseVersion();
-  await exec(`npm version ${newVersion}`);
+  const version = await getNewReleaseVersion();
+  await bumpVersion(version);
+}
+
+async function bumpVersion(version) {
+  console.log(`bumping version: ${version}`);
+  await exec(`npm version ${version} -m "[version bump] %s"`);
```

Comment:
Inspired from Bejamin's changes in ui-kit, I added this message so that the deployment pipeline would not flag version bump commits.

Ref: https://coveord.atlassian.net/wiki/spaces/CM/pages/1786708612/Package+Rollout+Certifier

Reply:
oh that's cool that we have a solution for these commits